packages/simple-xcm/simplexcm.ts

```diff
@@ -253,25 +257,39 @@ export class SimpleXcm {
     }
   }
 
-  async estimateExtrinsicXcmFees(
+  async tryEstimateExtrinsicXcmFees(
```

Comment:
The `try` prefix should be added to the `Estimator.estimateExtrinsicFees` methods instead.

```suggestion
  async estimateExtrinsicXcmFees(
```